mmvstr.c

```diff
@@ -43,8 +43,7 @@ long g_startTempAllocStack = 0;    /* Where to start freeing temporary allocatio
                                     special nested vstring functions) */
 void *tempAllocStack[MAX_ALLOC_STACK];
 
-static void freeTempAlloc(void)
-{
+void freeTempAlloc(void) {
```

Comment:
This changes visibility during linking

Reply:
Yes? It's public now

Reply:
Very unlikely, but it may distract previous calls to some other freeTempAlloc defined elsewhere.  You have to verify this detail, and I wasn't inclined to do so any more after having evaluated several thousand lines of code.  See my introductory note.

grep shows this is not the case.  So accepted.